jet-fuel/config/environments/production.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.configure do
+  config.cache_classes = true
+  config.eager_load = true
+  config.consider_all_requests_local       = false
+  config.action_controller.perform_caching = true
+  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.